libbeat/docs/outputconfig.asciidoc

```diff
@@ -537,6 +572,17 @@ output:
 
 If `pretty` is set to true, events written to stdout will be nicely formatted. The default is false.
 
+===== bulk_max_size
+
+The maximum number of events to bulk into batches. If the Beat sends single events, the events
+are collected into batches. If the Beat publishes a large batch of events (larger than the value
+specified by `bulk_max_size`), the batch is split.
+
+Specifying a larger batch size can improve performance by lowering the overhead of sending events. 
+However big batch sizes can also increase processing times
+
+The default is 200.
```

Comment:
@urso I'm not exactly sure what I should say based on your feedback here. It sounds like the description works for logstash and elasticsearch, but for console and file output, events are written one after another (which makes sense). But it sounds like you're saying that they are also buffered based on the value specified by bulk_max_size. Can you clarify as to which parts of the next need to change here?